packages/paste-design-tokens/tokens/global/border-color.yml

```diff
@@ -74,8 +77,8 @@ props:
 
   # destructive borders
   color-border-destructive-darker:
-    value: "{!palette-red-90}"
-    comment: Destructive hover border color
+    value: "{!palette-red-80}"
+    comment: Darker destructive hover border color
```

Comment:
This should stay red-90 in the default theme.

Reply:
Corrected, and fixed the comment.